### PR TITLE
Fixup alpine setup issues and add core/light images from master branch

### LINKED ISSUE #47: Alpine: test fails "Call to flock failed"
**C3OS version:**
Current master, not released

**CPU architecture, OS, and Version:**

**Describe the bug**
Test failure: https://github.com/c3os-io/c3os/runs/7554332112?check_suite_focus=true

**To Reproduce**
N/A

**Expected behavior**

**Logs**
```
      <string>: {"level":"info","ts":1658989504.254293,"caller":"provider/bootstrap.go:96","msg":"Configuring VPN"}
      {"level":"info","ts":1658989504.3057752,"caller":"provider/bootstrap.go:96","msg":"Configuring VPN"}
      Provider c3os at /usr/bin/agent-provider-c3os had an error: Failed setup VPN: could not start svc: failed starting service: edgevpn.  * Call to flock failed: Resource temporarily unavailable
       * WARNING: edgevpn is already starting
       (exit status 1)
```

**Additional context**
Maybe a `/run` / `/var/run` issue?

### PR DESCRIPTION
**Pull request by mudler**
This PR is multi-fold:
- Fixes #47 
- Touches some parts of #44 , core/light images are built from master branch and split k3s binary install off from base images
- Updates golang-ci versions
- Pin golang version used during build (libp2p does not support golang 1.19 yet)

### CODE REVIEW

**Comment by mudler** [comment]
I'm not entirely satisfied by the solution - but better be resilent than failing.

File: .github/workflows/image-corelight.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: Build core/light images
+
```
Comment (mudler) [inline .github/workflows/image-corelight.yaml:2]: The workflows here are just to start pushing core/light images from master, not yet for releases

File: .github/workflows/image-framework.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+name: Build Framework images
```
Comment (mudler) [inline .github/workflows/image-framework.yaml:1]: ditto for fw images

File: Earthfile [hunk 3]
```diff
@@ -52,6 +54,37 @@ test:
...
+    ARG OS_LABEL
+    ENV OS_ID=$OS_ID
+    ENV OS_NAME=$OS_NAME
+    ENV OS_REPO=$OS_REPO
+    ENV OS_VERSION=$OS_VERSION
+    ENV OS_LABEL=$OS_LABEL
+
+    # update OS-release file
+    RUN envsubst >/etc/os-release </usr/lib/os-release.tmpl && \
+        rm /usr/lib/os-release.tmpl
```
Comment (paynejacob) [inline Earthfile:72]: Could we leave the template for images that extend core and lite.  It would make sense for them to override theses values.

**Review by paynejacob — APPROVED** [review]
Looks good, one NIT we probably want to deleted the k3s version arg from all the image files.

File: Earthfile (hunk 3 above)
Comment (mudler) [reply Earthfile:72]: right :+1:

Comment (mudler) [reply Earthfile]: done in https://github.com/c3os-io/c3os/pull/56/commits/b5d4b1bfd99c288aa55548b0885161d560d3d0da

**Pull request merged**